cmd/utils.go

```diff
@@ -627,7 +627,7 @@ func UserHomeDir(log *LoggingConfig) string {
 	return homeDir
 }
 
-func getConfigLabels(projectConfig ProjectConfig, filename string) (map[string]string, error) {
+func getConfigLabels(projectConfig ProjectConfig, filename string, config *RootCommandConfig) (map[string]string, error) {
```

Comment:
Instead of requiring the entire config, you only need `log *LoggingConfig` here.